src/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import { Box} from "@mui/system";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
-function App() {
+import Login from "./components/Login/Login";
+import Main from "./components/Main"
+
+import "./App.css";
+
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Box>
+      <BrowserRouter>
+        <Routes>
+          <Route path="login" element={<Login />} />
+          <Route path="/" element={<Main />} />
+        </Routes>
+      </BrowserRouter>
+    </Box>
```

Comment:
Why is the router inside a box?

Reply:
if i move box down this error shows up
`[undefined] is not a <Route> component. All component children of <Routes> must be a <Route> or <React.Fragment>`